crates/simulacrum/src/lib.rs

```diff
@@ -286,11 +291,25 @@ impl<R, S: store::SimulatorStore> Simulacrum<R, S> {
         let epoch_start_timestamp_ms = self.store.get_clock().timestamp_ms();
 
         let next_epoch_system_package_bytes = if config.include_system_packages {
-            BuiltInFramework::iter_system_packages()
+            let packages: Vec<_> = match config.system_packages_snapshot {
+                Some(snapshot_version) => match load_bytecode_snapshot(snapshot_version) {
+                    Ok(snapshot_packages) => snapshot_packages,
+                    Err(e) => {
+                        panic!(
+                            "Failed to load bytecode snapshot for version {}: {}",
+                            snapshot_version, e
+                        );
+                    }
+                },
+                None => BuiltInFramework::iter_system_packages().cloned().collect(),
```

Comment:
Let's not even offer the option to use the current system packages -- we should never use it.